### DISCUSSION TITLE
Using font-display swap for eicons

### DISCUSSION THREAD

**Discussion by Salimoh** [elementor/elementor#14925] [Developers Community]
Hi
I got this error in the lighthouse report for my website.
I think it is a good idea ti use font-display:swap for this font by default to meet Google expectations.
Thanks
![image](https://user-images.githubusercontent.com/31356594/118021235-40373100-b370-11eb-906a-55d63caf6286.png)

**Comment by AliBorsan** [comment]
I solved this problem after looong suffer with this font file.
 First you have to open and edit this file :
elementor/assets/lib/eicons/css/elementor-icons.css and add at the end of @font-face{
 ;here}
add this: font-display:swap;
sometimes doesn't work because it is also included in (frontend.php) and being written everytime , so the final solution is in:
elementor/includes/frontend.php /line:1402
This part:
font-weight:400;font-style:normal,font-display:}
should be made:
font-weight:400;font-style:normal,font-display:swap;}
and this will solve the problem of this font with google insights permanently.
® Credits reserved!
